### DISCUSSION TITLE
Is the flathub version officially supported ?

### DISCUSSION THREAD

**Discussion by VaZark** [musescore/MuseScore#23215] [General]
I found a flatpak version of Musescore here : https://flathub.org/apps/org.musescore.MuseScore

If the flatpak version is an official release, I would love to switch to the flatpak as it's kinda like AppImage but updates similar to any linux distro's native packager

**Comment by MarcSabatella** [comment]
No, it is not officially supported, and in fact is known to have issues on many systems due to being built with incorrect setting.  There has been some talk about maybe moving to Flatpak in the future if there is sufficient interest and it can be done in a way that works as well as the AppImage.

Meanwhile, while the AppImage doesn’t update *automatically*, it does update very *easily* by just running “mscore4portable update” from the command line (assuming you installed it normally to begin with via the “install” option).  This automatically checks for updates, downloads any necessary files, and updates the installation.

**Reply by VaZark** [reply]
An official flatpak version would be amazing. IMO, the biggest advantage over AppImage is that the application itself is isolated and the end-user experience is very close to the dev environments. So there'd be far fewer worries about dealing with distro-specific issues

Also, it's ergonomically easier for me to run a `flatpak update` once a week than keeping up the release schedule for each of my apps :laughing:

**Reply by MarcSabatella** [reply]
I don’t understand what you mean about the Flatpak build being “isolated”.  In what way is it significantly different from AppImage in that respect?

Anyhow, you don’t need to keep up with the MuseScore release schedule.  The automatic check for updates works fine.  So the next time you run MuseScore after an update, it will let you know an update is available.

As I said, the possibility of eventually supporting Flatpak is being considered.  For example, see https://github.com/musescore/MuseScore/issues/19722.
